source/tls.txt

```diff
@@ -0,0 +1,107 @@
+.. _spark-configure-tls:
+
+=================
+Configure TLS/SSL
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, authenticate
+
+Overview
+--------
+
+In this guide, you can learn how to configure :wikipedia:`TLS/SSL
+<Transport_Layer_Security>` to secure communications between the
+{+connector-long+} and your MongoDB deployment.
+
+To use TLS/SSL, your application and each of your Spark
+workers must have access to cryptographic certificates that prove their
+identity. Store the certificates in your JVM trust store and your JVM
+key store. You can configure access to these certificates through your Spark
+configuration file, or when launching a Spark job from the command line.
+
+Create a JVM Trust Store
+------------------------
+
+The JVM trust store saves certificates that securely identify other applications
+with which your application interacts. Using these certificates, your
+application can prove that the connection to another application is genuine and
+secure.
+
+Create a trust store with the `keytool <https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html>`__ command line tool provided as part of the
+JDK:
+
+.. code-block:: sh
+
+   keytool -importcert -trustcacerts -file <path to certificate authority file>
+           -keystore <path to trust store> -storepass <password>
+
+Create a JVM Key Store
+----------------------
+
+The JVM key store saves certificates that securely identify your application to
+other applications. Using these certificates, other
+applications can prove that the connection to your application is genuine and
+secure.
+
+Create a key store by using the `keytool
+<https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html>`__, or
+`openssl <https://www.openssl.org/docs/>`__ command line tools.
+
+Enable TLS/SSL
+--------------
+
+You can enable TLS/SSL for the connection to your MongoDB instance through the
+``tls`` parameter in your connection URI.
```

Comment:
You could link to a description or instructions for connection URI here.

I'm not sure if there's anything more specific than this?
https://www.mongodb.com/docs/upcoming/reference/connection-string/#find-your-connection-string